### PR TITLE
Update actions version [skip ci]

### PR DESCRIPTION
**Pull request by YanxuanLiu**
Update actions version for Node.js 20

### CODE REVIEW

**Comment by YanxuanLiu** [comment]
build

File: .github/workflows/add-to-project.yml
Comment (jlowe) [inline .github/workflows/add-to-project.yml:1]: 2024 copyrights

File: .github/workflows/clang-format.yml
Comment (jlowe) [inline .github/workflows/clang-format.yml:1]: 2024 copyrights

File: .github/workflows/signoff-check.yml
Comment (jlowe) [inline .github/workflows/signoff-check.yml:1]: 2024 copyrights

**Comment by YanxuanLiu** [comment]
build

**Comment by YanxuanLiu** [comment]
build

**Comment by GaryShen2008** [comment]
build

**Review by NvTimLiu — APPROVED** [review]

**Pull request merged**